library/include/hipblas.h

```diff
@@ -77,6 +77,25 @@
 #endif
 #endif
 
+// Macros used before hipBLAS version 3.0
+#define HIPBLAS_R_16F HIP_R_16F
```

Comment:
I would put it at end of file to ensure they aren't used again in this file.   
@daineAMD  should they be in an #ifndef __HIP_PLATFORM_NVCC__ as well

Reply:
Yeah good idea to move to the end. Seems like for this specific fix it could be excluded in cuda builds, not sure if something else will pop up, though.

Reply:
Or does HIP_R_16F get set up in cuda the same so fine for both?

Reply:
Oh, yeah, this shouldn't give any issues with cuda, those types exist there too. So we can leave it for simplicity.

Reply:
next commit moves definitions to the end of hipblas.h